core/src/test/java/nav/statusplattform/core/repositories/RecordRepositoryTest.java

```diff
@@ -290,4 +290,13 @@ private RecordEntity getRandomizedRecordEntity() {
                 .setResponsetime(SampleData.getRandomResponseTime());
     }
 
+    public static List<RecordEntity> getRandomizedRecordEntitiesForService(ServiceEntity serviceEntity) {
+        List<RecordEntity> records = new ArrayList<>();
+        int numberOfRecords = SampleData.randomNonZeroPositiveInteger(12);
+        for (int i = 0; i < numberOfRecords; i++) {
+            records.add(SampleData.getRandomizedRecordEntityForService(serviceEntity));
+        }
+        return records;
+    }
```

Comment:
Renamed to describe what class is actually doing; Additionally, Access modifier changed from public static because the method is used solely within the class; Finally, added argument so value may be altered depending on the test criteria.